### ISSUE TITLE
Populate Event details for Work-level Events

### ISSUE THREAD

**Issue by eporter23** [emory-libraries/dlp-curate#1030]
See [wireframe - section 6](https://docs.google.com/drawings/d/1_86XPEqC1XKKv6nNdsUPYYF0suKAkR3V1ZTrevOgJOw/edit)

Adds work-level Event details listing to the new View Preservation Event Detail page

**Comment by devanshu-m** [comment]
Output:
![image.png](https://images.zenhubusercontent.com/5c12bc2dd3542818d52aa3a0/9026ba56-4294-4799-a797-6fdbb4188e1c)

**Comment by eporter23** [comment]
@devanshu-m this screenshot looks great as far as styling goes. We probably won't need any new work to be done for #1031.

**Comment by kmichaelis** [comment]
Metadata extraction is shown as a work-level event in the wireframe, but does not show up on any works I have looked at in test.

**Comment by kmichaelis** [comment]
This might be a missing-events issue similar to #1246. We will wait for Emily's feedback.

**Comment by eporter23** [comment]
@kmichaelis @devanshu-m here is recently updated [documentation on the preservation events](https://docs.google.com/document/d/16gYBYO5f1GLV3Wl9MjI0vRoN3wSejLQaxFvzd3Qua2I/edit) that have been implemented. We made some changes that may not be reflected in the wireframe, but for work-level events, we should see Policy Assignment and Validation only. We weren't able to get Metadata Extraction  working as hoped.
